api/envoy/config/core/v3/health_check.proto

```diff
@@ -186,6 +186,19 @@ message HealthCheck {
     // the :ref:`hostname <envoy_v3_api_field_config.endpoint.v3.Endpoint.HealthCheckConfig.hostname>` field.
     string authority = 2
         [(validate.rules).string = {well_known_regex: HTTP_HEADER_VALUE strict: false}];
+
+    // Specifies a list of HTTP headers that should be added to each request that is sent to the
```

Comment:
Should we refer to this as "initial metadata" as that is the gRPC terminology for this?

Reply:
Good point, reworded the comment, please take another look.